analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -73,45 +105,38 @@ public void Disposed_Using_WithDeclaration()
     public void Disposed_Using_WithExpressions()
     {
         var d = new Disposable();
-        using (d) // FIXME Non-compliant
+        using (d) // FN
         {
             d.Dispose();
         }
     }
 
-    public void Disposed_Using_Parameters(IDisposable param1)
-    {
-        param1.Dispose();
-        param1.Dispose(); // FIXME Non-compliant
-    }
-
-    public void Close_ParametersOfDifferentTypes(IInterface1 interface1, IDisposable interface2)
+    public void Close_ParametersOfDifferentTypes(IWithDispose withDispose, IDisposable disposable)
     {
-        // Regression test for https://github.com/SonarSource/sonar-dotnet/issues/1038
```

Comment:
this link points to an issue from 2017 - so I guess the 
OLD engine. 
I don't see a reason to keep them here - if I'm wrong I'll revert.

Reply:
It tests reasonable logic that makes sense to test. So we should keep it

Reply:
We don't need the `Regression test for`, the link itself is enough